google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -48,8 +48,8 @@ appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
 appengine.cloudsdk.location.badchars.message=The specified Cloud SDK directory contains invalid characters or starts or ends with spaces.
 appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in the specified directory.
+appengine.cloudsdk.location.invalid.message.deploy=The Cloud SDK is misconfigured. To fix, reconfigure the Server above.
```

Comment:
Yeah, that's where I wanted to go with adding deploy at the end. Though it might not be too much of a hint.

I'll rename it to appengine.cloudsdk.location.invalid.message.deploymentconfiguration, hopefully it's clearer?